packages/docs/docs/dev/connectors.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Wallet Connectors
+category: For Developers
+---
+
+# Wallet Connectors
+
+Wallet Connectors is a interface to integrate with Thirdy Party Wallets on the Fuel Ecosystem. That expose all the current installed wallets in a unifinied maner
+via `window.fuel`.
+
+<Examples.Connectors />
+
+## Methods
+
+### Listing connectors
+
+To list all available connectors you can use the `listConnectors()` method.
+
+<CodeImport
+  file="../../examples/Connectors.tsx"
+  commentBlock="listConnectors"
+/>
+
+### Selecting connector
+
+For selecting a different Wallet Connector use the `selectConnector(connectorName: string)` method.
+
+<CodeImport
+  file="../../examples/Connectors.tsx"
+  commentBlock="selectConnector"
+/>
+
+## Events
+
+### CurrentConnector
+
+This event is triggered when the Current Connector is change. The callback function receives the `FuelWalletConnector` object.
+
+<CodeImport
+  file="../../examples/Connectors.tsx"
+  commentBlock="eventCurrentConnector"
+/>
+
+### Connectors
+
+This event is triggered when the Connectors list is updated. The callback function receives the list of connectors, `Array<FuelWalletConnector>`.
+
+<CodeImport
+  file="../../examples/Connectors.tsx"
+  commentBlock="eventConnectors"
+/>
+
+## For Wallet Developers
+
+Wallet Connector is used for Thirdy Party Wallet devleoped by the community to integrate with the current fuel instance on the window, enabling a better experience for
+developers and users.
+
+### Creating a Wallet Connector
+
+Inside the `pageScript.ts` you can call the method `createConnector` instead of creating a new object on the window. This method will add you Connector to the current,
+`window.fuel` element.
+
+```ts
+import { createConnector } from '@fuel-wallet/sdk';
+
+createConnector({ name: 'Wallet Name' });
+```
+
+In case you are using our Commnucation SDK, [Conent Page Script](https://developer.chrome.com/docs/extensions/mv3/content_scripts/) import `ContentProxyConnection` and inform the name of Wallet Exntesion.
```

Comment:
```suggestion
In case you are using our Communication SDK, [Conent Page Script](https://developer.chrome.com/docs/extensions/mv3/content_scripts/) import `ContentProxyConnection` and inform the name of Wallet Exntesion.
```